src/lib.rs

```diff
@@ -29,6 +29,31 @@ pub extern "C" fn map_hello_world(block_ptr: *mut u8, block_len: usize) {
     }
 }
 
+#[no_mangle]
+pub extern "C" fn map_mint(block_ptr: *mut u8, block_len: usize) {
+    substreams::register_panic_hook();
+
+    let blk: pb::eth::Block = proto::decode_ptr(block_ptr, block_len).unwrap();
+
+    for trx in blk.transaction_traces {
+        for call in trx.calls {
+            for log in call.clone().logs {
```

Comment:
You should try to avoid those `clone` when possible as their are doing a deep clone of the call which is not required here. 

You can check the latest `substreams-template` code which removed it.